toxcore/tox.h

```diff
@@ -837,15 +834,16 @@ typedef enum Tox_Err_New {
     TOX_ERR_NEW_NULL,
 
     /**
-     * The function was unable to allocate enough memory to store the events_alloc
-     * structures for the Tox object.
+     * The function was unable to allocate enough memory to store the
+     * events_alloc structures for the Tox object.
```

Comment:
"internal", maybe check the rest of the file(s) for having that mistake.

Reply:
Should be fixed now, please double check.